src/Microsoft.Bcl.HashCode/src/Microsoft.Bcl.HashCode.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+      <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+      <StrongNameKeyId>Open</StrongNameKeyId>
+      <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+      <Nullable>enable</Nullable>
+      <AssemblyVersion>1.0.1.0</AssemblyVersion>
+      <!-- Bump the below version each time we need to service a new version and ensure GeneratePackageOnBuild is set to true -->
+      <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+      <ServicingVersion>2</ServicingVersion>
+  
+      <!-- Package version properties -->
+      <VersionPrefix>1.1.$(ServicingVersion)</VersionPrefix>
+      <PackageValidationBaselineVersion>1.1.$([MSBuild]::Subtract($(ServicingVersion), 1))</PackageValidationBaselineVersion>

```

Comment:
Are these the right version prefix and package validation baseline versions to use? According to https://github.com/dotnet/corefx/blob/b86ce69c316b4b088356b89bab80f7fa79ff799c/pkg/Microsoft.Private.PackageBaseline/packageIndex.json#L36-L48 , and by attempting to imitate what was done by @joperezr in System.Buffers, The latest version available in `StableVersions` is 1.1.1. Notice it is not the same as what is being used in `BaselineVersion` or `AssemblyVersionInPackageVersion`, so I'm not sure.